File: Programming-languages-and-laboratory/Python/Exercises/22_compito_java_20180622/Teoria.py
from Corso import Corso


class Teoria(Corso):
    def __init__(self, codice, tipo, nome, nome_cognome_docente, codice_aula, ore_settimanali, ore_lezione):
        super().__init__(codice, tipo, nome, nome_cognome_docente)
        self._codice_aula = codice_aula
        self._ore_settimanali = ore_settimanali
        self._ore_lezione = ore_lezione

    def __str__(self):
        return super().__str__() + "\t" + self._nome_cognome_docente + "\t" + "teoria\t" + str(self._codice_aula) + "\t" + str(self._ore_settimanali) + "\t" + str(self._ore_lezione) + "\t-\t-\t-"

File: II - Linguaggi di programmazione e laboratorio/Esame/Java/22_compito_java_2018_06_22/Teoria.java
public class Teoria extends Corso {
	private int codiceAula, oreSettimanali;
	private double oreLezione;
	
	public Teoria(int codiceCorso, String tipo, String nomeCorso, String nomeCognomeDocente, int codiceAula, int oreSettimanali, double oreLezione) {
		super(codiceCorso, tipo, nomeCorso, nomeCognomeDocente);
		this.codiceAula = codiceAula;
		this.oreSettimanali = oreSettimanali;
		this.oreLezione = oreLezione;
	}
	
	public String toString() {
		return nomeCorso + "\t" + codiceCorso + "\t" + nomeCognomeDocente + "\t" + tipo + "\t" + codiceAula + "\t" + oreSettimanali + "\t" + oreLezione + "\t-\t-\t-";
	}
}